website/project/__init__.py

```diff
@@ -6,6 +6,7 @@
 from framework.mongo.utils import from_mongo
 from modularodm import Q
 from website.exceptions import NodeStateError
+from website.util.sanitize import strip_html
```

Comment:
From the COSDev style guide, 
*"Import entire modules instead of individual symbols within a module"*

http://cosdev.readthedocs.org/en/latest/style_guides/python.html#imports

This is mainly relevant for the case of `strip_html` because we have so many different sanitizers, making the name more confusing than it would otherwise be. Writing `sanitize.strip_html` makes it a bit clearer which sanitizer is being used.